ListViewMaui/ListViewMaui/Platforms/iOS/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using ObjCRuntime;
+using UIKit;
+
+namespace ListViewMaui;
+
+public class Program
+{
+	// This is the main entry point of the application.
+	static void Main(string[] args)
+	{
+		// if you want to use a different Application Delegate class from "AppDelegate"

```

Comment:
// If you want to use a different Application Delegate class from "AppDelegate,"